src/main/java/com/rhombus/api/examples/UserList.java

```diff
@@ -0,0 +1,146 @@
+
+        final CommandLine commandLine;
+        try
+        {
+            commandLine = new DefaultParser().parse(options, args);
+        }
+        catch (ParseException e)
+        {
+            System.err.println(e.getMessage());
+
+            new HelpFormatter().printHelp(
+                    "java -cp rhombus-api-examples-all.jar com.rhombus.api.examples.UserList",
+                    options);
+            return;
+        }
+
+        String apikey = commandLine.getOptionValue("apikey");
+        String csvName;
+        String reportName;
+
+        if (commandLine.hasOption("csv"))
+        {
+            csvName = commandLine.getOptionValue("csv");
+        }
+        else
+        {
+            csvName = "csvFile.csv";
+        }
+
+        if (commandLine.hasOption("path"))
+        {
+            reportName = commandLine.getOptionValue("path");
+        }
+        else
+        {
+            reportName = System.getProperty("user.dir");;
+        }
+
+        _initialize(apikey);
+        UserGetUsersInOrgWSResponse UserData = getUserdata();
+        File fileCSV = CSVCreate(reportName, csvName);
+        CSVAdd(UserData, fileCSV);
+    }
+
+    private static void _initialize(final String apiKey) throws Exception
+    {
+        /*
+         * API CLIENT
+         */
+        {
+            final List<Header> defaultHeaders = new ArrayList<>();
+            defaultHeaders.add(new BasicHeader("x-auth-scheme", "api-token"));
+            defaultHeaders.add(new BasicHeader("x-auth-apikey", apiKey));
+
+            final ClientBuilder clientBuilder = ClientBuilder.newBuilder();
+            clientBuilder.register(new JacksonJaxbJsonProvider().configure(SerializationFeature.FAIL_ON_EMPTY_BEANS, false).configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false));;
+
+            final HostnameVerifier hostnameVerifier = new HostnameVerifier() {
+                @Override
+                public boolean verify(String hostname, SSLSession session) {
+                    return true;
+                }
+            };
+
+            clientBuilder.hostnameVerifier(hostnameVerifier);
+
+            _apiClient = new ApiClient();
+            _apiClient.setHttpClient(clientBuilder.build());
+            _apiClient.addDefaultHeader("x-auth-scheme", "api-token");
+            _apiClient.addDefaultHeader("x-auth-apikey", apiKey);
+
+            _userwebservice = new UserWebserviceApi(_apiClient);
+        }
+    }
+
+    private static UserGetUsersInOrgWSResponse getUserdata() throws Exception
+    {
+        final UserGetUsersInOrgWSRequest userRequest = new UserGetUsersInOrgWSRequest();
+        final UserGetUsersInOrgWSResponse userResponse = _userwebservice.getUsersInOrg(userRequest);
+        return userResponse;
+    }
+
+    private static File CSVCreate(String reportName, String csvName)
+    {
+        File csvFile = new File(reportName + "/" + csvName);
```

Comment:
make sure that you are handling the case where the file _can't_ be opened gracefully.  Likely a try/catch block here.